src/lib/Importer/SchemaImporter.php

```diff
@@ -77,12 +77,6 @@ private function importSchemaTable(
     ): void {
         $table = $schema->createTable($tableName);
 
-        if (!empty($defaultTableOptions)) {
-            foreach ($defaultTableOptions as $name => $value) {
-                $table->addOption($name, $value);
-            }
-        }
-
```

Comment:
This is a bug which requires either separate commit to be merged fast-forward, or (better) a separate PR. What needs to be investigated is the source of the error. I have a feeling that we can configure those default options somewhere in Ibexa by means of semantic config, but they're not respected. Had no time to investigate it yet.

Reply:
I can remove this change, but from my perspective that is dead code.

And tbh I don't see any valid default options that would entail **all** tables. Collation & similar should be declared on database level, if needed.